docs/content/docs/reference/nextjs/index.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Introduction'
+---
+
+It is possible to use [`@nhost/react`](/reference/react) in any Next.js page that would be configured to render on the client side.
+
+When rendering a page from the server side, Next.js needs to get some information from the client to determine their authentication status. Such a communication is only available from cookies, and the the Nhost client has been designed to enable such a mechanism.
```

Comment:
"It is possible to use [`@nhost/react`](/reference/react) in any Next.js page that would be configured to render on the client-side.

When rendering a page from the server-side, Next.js needs to get some information from the client to determine their authentication status. Such communication is only available from cookies, and the Nhost client is designed to enable such a mechanism."